src/mmw/bundle.sh

```diff
@@ -115,6 +116,7 @@ JS_DEPS=(backbone
          ./js/shim/shutterbug.js
          ./js/shim/highstock.js
          ./js/shim/nv.d3.js
+         ./js/shim/tableExport.min.js
```

Comment:
Turns out `tableExport.min.js` is a fork of that repo that was made into a jQuery plugin. It doesn't have a corresponding npm package.